jerry-debugger/jerry_client_main.py

```diff
@@ -149,6 +149,17 @@ def arguments_parse():
     return args
 
 
+if sys.version_info.major >= 3:
+    # pylint: disable=invalid-name
+    _ord_orig = ord
+    def _ord_compat(c):
+        if isinstance(c, int):
+            return c
+        return _ord_orig(c)
+    # pylint: disable=redefined-builtin
+    ord = _ord_compat
```

Comment:
Was wondering which one would be better:
A) Having this ord compatibility paching or
B) Having a new method (`safe_ord`?) which replaces all occurrences of the `ord` calls in the python code and does the same as this compatibility method.

In case of B we could also remove these newly added pylint disable comments.

Also, the `jerry_client_rawpacket.py` also needs the same change.

Reply:
I've changed it to use a new `safe_ord` function as you've suggested.